### PR TITLE
Add Polaris editions page

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: app/templates/editions/polaris.hbs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{page-title "Polaris"}}
+
+<section class="container" aria-labelledby="octane-edition">
+  <h1 id="octane-edition">The Polaris Edition of Ember</h1>
+  <p class="intro">
+    We are now full steam ahead on getting the next edition of Ember completed! We will be updating this page with relevant information so make sure to keep an eye out for news.
```
Comment (MinThaMie) [inline app/templates/editions/polaris.hbs:6]:
```suggestion
    We are now full steam ahead on getting this next edition of Ember completed! We will be updating this page with relevant information so make sure to keep an eye out for news.
```

File: app/templates/editions/polaris.hbs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    Shipping an edition is a collaborative effort that involves the entire Ember community.
+    If you are able and interested in contributing, check out the <a href="https://github.com/orgs/emberjs/projects/9/views/1">main project board</a>.
+  </p>
+
+  <h2>Frequently Asked Questions</h2>
+  <dl class="dl-faq">
+    <dt>Where can I get help or ask questions?</dt>
+    <dd>
+      Visit <LinkTo @route="community">the community page</LinkTo> to join the forums or chat!
+      Discord has channels specifically for Octane where you can search for other people's questions too. Read the "welcome" and "setup profile" channels to learn how to get posting permissions.
```
Comment (MinThaMie) [inline app/templates/editions/polaris.hbs:25]: Maybe add the link to our discord server just like you did with the forum

**Review by MinThaMie — APPROVED** [review]
Love it! Just some minor comments. I would say ship it!

**Review by jenweber — COMMENTED** [review]
I like the strategy of getting this out ASAP once there is an image. Thank you!

**Comment by MinThaMie** [comment]
Discussed during the learning meeting to use the blueprint Tomster until we have a real image :)

**Comment by clionaderoiste** [comment]
❤️

**Pull request merged**